Dart/resources/postfixTemplates/DartRemotePostfixTemplate/description.html

```diff
@@ -0,0 +1,5 @@
+<html>
```

Comment:
I'm getting an error because `before.dart.template` and `after.dart.template` are missing next to this file. I've added empty ones, let me know if you meant smth else.